src/components/Order/Stepper.jsx

```diff
@@ -3,24 +3,26 @@ import Box from "@mui/material/Box";
 import Stepper from "@mui/material/Stepper";
 import Step from "@mui/material/Step";
 import StepLabel from "@mui/material/StepLabel";
-import Button from "@mui/material/Button";
 import useStep from "./useStep";
-import {containerStyle,stepContentBoxStyle ,buttonContainerStyle,
-         spacerStyle,backButton} from "./Stepper.styles";
-         import { UI_TEXT } from "../../constants/text";
+import {
+  containerStyle, stepContentBoxStyle, buttonContainerStyle,
+  spacerStyle, backButton
+} from "./Stepper.styles";
+import { UI_TEXT } from "../../constants/text";
 
 import OrderSummary from "./steps/OrderSummary";
 import CustomerDetails from "./steps/CustomerDetails";
 import PaymentDetails from "./steps/PaymentDetails";
 import OrderComplete from "./steps/OrderComplete";
+import SharedButton from "../shared/button";
 
 const stepComponents = [
-    <OrderSummary />,
-    <CustomerDetails />,
-    <PaymentDetails />,
-    <OrderComplete />
-  ];
-  
+  <OrderSummary />,
+  <CustomerDetails />,
```

Comment:
Why do you have prettier differences? 
Didn't we configure a prettier for everyone?

Reply:
@OrtalNisim  So there prettier does not work? and you are using your local prettier? what causes this spaces difference?